server/routerlicious/packages/services-core/src/tenant.ts

```diff
@@ -17,6 +17,12 @@ export interface ITenantConfig {
 
 	customData: ITenantCustomData;
 
+	// Indicates if keyless access is enabled for this tenant.
+	enableKeylessAccess: boolean;
+
+	// Indicates if key access is enabled for this tenant.
+	enableKeyAccess: boolean;
```

Comment:
Since this is not yet used anywhere, would you please add a comment to this stating so?

```ts
/**
 * Indicates if (shared secret) key access is enabled for this tenant.
 * @remarks
 * This value is never read and cannot be updated via public APIs in Routerlicious.
 */
```

Also, this interface doesn't use the correct doc-comment syntax. Would you mind updating that while you're here?